### ISSUE TITLE
Should be using `if let` instead of `match` with empty arm

### ISSUE THREAD

**Issue by HCastano** [paritytech/wasm-utils#85]
I've been digging through the repo and I've seen a lot of code that looks like the following:

```rust
match element {
    Some(i) => {
        // Do thing
    },
    _ => { }
};
```

This pattern can be simplified using `if let`. For the code above it would look like this:

```rust
if let Some(i) = element {
    // Do thing
}
```

This wouldn't change how the code performs, but it would increase readability throughout the codebase. I'm more than happy to work on a PR to refactor `match` statements of this type into `if let` statements if that's something that you would like to see.

**Comment by NikVolf** [comment]
Sure, feel free to submit PR :)

**Comment by HCastano** [comment]
Closed by #87
